src/containers/ReviewsIWrote/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import { get, flatten } from 'lodash'
+// redux
+import { connect } from 'react-redux'
+import { compose } from 'recompose'
+// components
+import Reviews from 'components/Reviews'
+// hoc
+import withTransactions from 'containers/Hoc/withTransactions'
+// helpers
+import { getAddress } from 'helpers/wallet'
+
+class ReviewsIWrote extends Component {
+
+  render() {
+    const { wallet, reviews, loading, error } = this.props
+    const address = getAddress(wallet)
+
+    let transformedReviews = Object.values(reviews)
+      .filter(r => r && !r.loading && !r.error && r.customer_address === address)
+    console.log(transformedReviews)
+    transformedReviews = transformedReviews.map(r => ({
+        subject: {
+          name: 'Vendor'
+        },
+        author: {
+          name: 'You'
+        },
+        platform: {
+          name: 'chlu',
+          url: 'chlu.io'
+        },
+        review: {
+          text: r.review_text
+        },
+        rating: {
+          min: 1,
+          value: r.rating,
+          max: 5
+        },
+        verifiable: true,
+        proof: true
```

Comment:
The proof should be the signatures from the vendor marketplace key and the popr. You used to do that in the old demo, can we bring them back? Or want to leave it for later?

Reply:
I think we should leave this for later and only show that they are verifiable and also verified, since reviews will show as an error if they aren't valid